web/themes/new_weather_theme/new_weather_theme.libraries.yml

```diff
@@ -69,6 +69,23 @@ radar:
         defer: true
         data-wx-radar-cmi: true
 
+leaflet:
+  version: 1.9.4
+  css:
+    theme:
+      assets/css/leaflet.css:
+        attributes:
+          integrity: sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=
+          crossorigin: /
+  js:
+    https://unpkg.com/leaflet@1.9.4/dist/leaflet.js:
+      data-wx-leaflet: true
+    https://unpkg.com/esri-leaflet@3.0.12/dist/esri-leaflet.js:
+      data-wx-leaflet-esri: true
+    https://unpkg.com/esri-leaflet-vector@4.2.3/dist/esri-leaflet-vector.js:
+      data-wx-leaflet-esri-vector: true
+    assets/js/components/alertMap.js:
+
```

Comment:
It's not strictly necessary, but you _could_ add the `defer` attribute here I think

Reply:
`defer` unblocks page rendering, right? That's probably a good thing to do!